test/unit/sth_test.js

```diff
@@ -969,6 +969,110 @@ describe('Data removal', function() {
   );
 });
 
+describe('POST /admin/log', function() {
```

Comment:
All this cases differ only in a single string. Please, create a function generator with that string as a parameter to encapsulate all this code.